### ISSUE TITLE
`NOW()` not recognized as `now()`

### ISSUE THREAD

**Issue by dargueta** [xnuinside/omymodels#18]
**Describe the bug**

`NOW()` is not recognized as equivalent to `now()` when used as a default value, even though SQL is case-insensitive.

**To Reproduce**

Given the following SQL:

```py
ddl = """
CREATE TABLE asdf (
  t TIMESTAMP DEFAULT NOW()
);
"""
print(omymodels.create_models(ddl=ddl, models_type="dataclass")["code"])
```

The output is

```py
import datetime
from dataclasses import dataclass


@dataclass
class Asdf:

    t: datetime.datetime = 'NOW()'  # Default value is wrong
```

Note that, while the datatype of the column is correct, the default value is a string instead of a call to `datetime.datetime.now(). If we make `NOW()` lowercase, we get the expected output.

```py
import datetime
from dataclasses import dataclass


@dataclass
class Asdf:

    t: datetime.datetime = datetime.datetime.now()
```

n.b. the expected output has a bug in it which I'll file a separate ticket for

**Expected behavior**

SQL is case-insensitive, therefore `now()` and `NOW()` should behave identically and produce the same output.

**Additional context**

Python: 3.8.2
Version: 0.8.1

**Comment by xnuinside** [comment]
@dargueta thanks for the report, I will fix it as soon as possible

**Comment by xnuinside** [comment]
@dargueta  fix also in 0.8.3 - https://pypi.org/project/omymodels/0.8.3/. Test - https://github.com/xnuinside/omymodels/blob/main/tests/test_dataclasses.py#L200

**Comment by dargueta** [comment]
Thanks!
